.github/stale.yml

```diff
@@ -11,7 +11,14 @@ staleLabel: wontfix
 # Comment to post when marking an issue as stale. Set to `false` to disable
 markComment: >
   This issue has been automatically marked as stale because it has not had
-  recent activity. It will be closed if no further activity occurs. Thank you
-  for your contributions.
+  recent activity. It will be closed if no further activity occurs.
+  Please note: We are just a few people who contribute to a shared project,
+  it's impossible for us to fix every bug with such limited resources.
+  If this is important to you for your business then please talk with your
+  Red Hat, Canonical or SUSE technical account manager and file a bug in
```

Comment:
I think rather than enumerate distros, bug trackers and priorities, it should be something like this.
"If this is important to you for your business please talk with your technical account manager about arranging resources to solve this issue".

Reply:
Ack, thanks.